packages/webapp/src/components/CreateUserAccount/index.jsx

```diff
@@ -22,31 +23,69 @@ export default function PureCreateUserAccount({ onSignUp, email, onGoBack }) {
   } = useForm({
     mode: 'onTouched',
   });
+
   const NAME = 'name';
   const GENDER = 'gender';
+  const LANGUAGE = 'language';
   const BIRTHYEAR = 'birth_year';
   const PASSWORD = 'password';
   const password = watch(PASSWORD, undefined);
   const { t } = useTranslation(['translation', 'common', 'gender']);
   const title = t('CREATE_USER.TITLE');
+
   const {
     isValid: isPasswordValid,
     hasNoSymbol,
     hasNoDigit,
     hasNoUpperCase,
     isTooShort,
   } = validatePasswordWithErrors(password);
+
   const genderOptions = [
     { value: 'MALE', label: t('gender:MALE') },
     { value: 'FEMALE', label: t('gender:FEMALE') },
     { value: 'OTHER', label: t('gender:OTHER') },
     { value: 'PREFER_NOT_TO_SAY', label: t('gender:PREFER_NOT_TO_SAY') },
   ];
 
+  const languageOptions = [
+    { value: 'en', label: t('PROFILE.ACCOUNT.ENGLISH') },
+    { value: 'es', label: t('PROFILE.ACCOUNT.SPANISH') },
+    { value: 'pt', label: t('PROFILE.ACCOUNT.PORTUGUESE') },
+    { value: 'fr', label: t('PROFILE.ACCOUNT.FRENCH') },
+  ];
+
+  const getLanguageOption = (language) => {
```

Comment:
can add option as a key in the { value: 'en', label: t('PROFILE.ACCOUNT.ENGLISH') },

e.g { value: 'en', label: t('PROFILE.ACCOUNT.ENGLISH'), option: 0 },

Reply:
Why do we need a key here?

Reply:
const getLanguageOption = (language) => {
    switch (language) {
      case 'en':
        return 0;
      case 'es':
        return 1;
      case 'pt':
        return 2;
      case 'fr':
        return 3;
    }
  };

  you could have tried key value pairs of language 

  e.g: 
getLanguageOption = (language) => {
  const options = {
    'en': 0,
    'es': 1,
    'pt': 2,
    'fr': 2,
  }
  return options[language]
}

Reply:
const languageOptions = [
  { 
    value: 'en', 
    label: t('PROFILE.ACCOUNT.ENGLISH')
  },
  { value: 'es', label: t('PROFILE.ACCOUNT.SPANISH') },
  { value: 'pt', label: t('PROFILE.ACCOUNT.PORTUGUESE') },
  { value: 'fr', label: t('PROFILE.ACCOUNT.FRENCH') },
];


Also, you can use index position to get the values as 0, 1, 2, 3

Reply:
const languageOptions = [
{ value: 'en', label: 'english'},
{ value: 'es', label: 'spanish' },
{ value: 'pt', label: 'Portuguese ' },
{ value: 'fr', label: 'french' },
];

const getLanguageOption = (language) => {
return languageOptions.findIndex(object => object.value === language);
}

console.log(getLanguageOption('en'))